Templates/Specimens_Metadata_2021Q1_template.tsv

```diff
@@ -0,0 +1,18 @@
+Column Name	Description	Ontology identifier	Ontology term name	Ontology Description
+Specimen ID	Unique identifier for this specimen	OBI:0100051	specimen	A material entity that has the specimen role.
+Parent specimen ID	Identifier for this specimen’s parent specimen (optional)	OBI:0100051	specimen	A material entity that has the specimen role.
+Number of subspecimens	A count of specimens derived from this specimen (cumulative)	APOLLO_SV:00000032	count	A measurement datum that is the output of counting.
```

Comment:
Is this transitive or direct? The number may not be known at time of deposition, and can change.

Reply:
It is important to separate the notion of "estimate" and "report" number of  sub-subpecimens and the true number.
If this an "estimate"/"report" the description should reflect that

Reply:
Asserted number of sub-specimens is quite useful as a sanity check against the computed number.

Reply:
I changed this to `Asserted number of subspecimens`. I'm still not sure whether this should be cumulative or not. Ideally all the subspecimens would have entries here so they could be calculated instead of manually inputted. The case that I worry about is when the number of cells is estimated or manually counted in population imaging studies.